.github/workflows/command-fmt.yml

```diff
@@ -37,6 +37,8 @@ jobs:
           GH_TOKEN: ${{ github.token }}
       - name: Run FMT
         run: |
+          cargo +nightly fmt
```

Comment:
I sometimes get different results still in my machine and CI, is this often because of different nightly versions? might be good to print the exact nightly version, any/or anything else needed to reproduce, in the error report phase if possible.

Reply:
good point, added